test/release/examples/primers/distributions.chpl

```diff
@@ -36,22 +36,24 @@ config const n = 8;
 //
 const Space = {1..n, 1..n};
 
-// Block (and distribution basics)
-// -------------------------------
+// blockDist (and distribution basics)
+// -----------------------------------
 //
-// The ``Block`` distribution distributes a bounding box from
+// The ``blockDist`` distribution distributes a bounding box from
 // n-dimensional space across the target locale array viewed as an
-// n-dimensional virtual locale grid.  The bounding box is blocked
-// into roughly equal portions across the locales.  Note that domains
-// declared over a Block distribution can also store indices outside
-// of the bounding box; the bounding box is merely used to compute
-// the blocking of space.
-//
-// In this example, we declare a 2-dimensional Block-distributed
-// domain ``BlockSpace`` and a Block-distributed array ``BA`` declared over
-// the domain.
-//
-const BlockSpace = Space dmapped Block(boundingBox=Space);
+// n-dimensional grid of locales.  The bounding box is blocked into
```

Comment:
Should this read "distributes a bounding box from a n-dimensional grid of locales"?

Reply:
I don't think so, but your question made me go back and re-read and try to streamline that whole paragraph.  I've currently got:

> // The ``blockDist`` distribution partitions an n-dimensional bounding          
// box between a set of target locales arranged into a conceptual               
// n-dimensional grid.  The bounding box is divided into roughly                
// equal-size block sections, where each locale owns one of them.               
// Note that domains declared using a ``blockDist`` distribution can            
// also store indices outside of the bounding box; the bounding box is          
// merely used to compute a partitioning of n-dimensional space, not            
// to constrain legal domains.                                                  

Thoughts?

Reply:
That reads better to me

Reply:
I accidentally checked this in with a batch of other test updates but didn't call it out specifically.